site/content/en/docs/install/install.md

```diff
@@ -142,3 +146,113 @@ These are the ports used by Envoy Gateway and the managed Envoy Proxy.
 {{% alert title="Next Steps" color="warning" %}}
 Envoy Gateway should now be successfully installed and running, but in order to experience more abilities of Envoy Gateway, you can refer to [Tasks](../tasks).
 {{% /alert %}}
+
+{{% /tab %}}
+{{% tab header="Install with Kubernetes YAML" %}}
+In this guide, we'll walk you through installing Envoy Gateway in your Kubernetes cluster.
+
+The manual install process does not allow for as much control over configuration
+as the [Helm install method](./install-helm), so if you need more control over your Envoy Gateway
+installation, it is recommended that you use helm.
+
+## Before you begin
```

Comment:
I think we can move this paragraph out of tabpane

Reply:
Hi @zirain I tried to move this out of tabpane but it became static and was not able to switch with the switching tabs. Should I attach a preview of it after making it out of tabpane?

Reply:
`Version Compatibility Matrix` part should be moved out of tabpane.